sdk/core/Azure.Core/samples/ProtocolMethods.md

```diff
@@ -2,11 +2,19 @@
 
 ## Introduction
 
-Most Azure SDK Clients expose methods that take 'model' parameters, C# classes which map to the request or response content of the REST call.
+Azure SDK clients provide an interface to Azure services by translating API requests to REST API calls. When there's a schematized body in the request or response we call this a 'model' in the REST API.
```

Comment:
I would not call it "API requests". Maybe "by translating library calls to REST requests."

Reply:
Also, do we really call it "model" in REST APIs or we call it "model" in library APIs?

Reply:
Unsure, I was taking @annelo-msft 's advice and trying to give things names to make it more clear. 

Is there another term here that makes it more clear?

Reply:
That was probably me being sloppy with my words - apologies.  Precise terminology will be found in Jeff R's [Designing HTTP APIs](https://www.youtube.com/watch?v=9Ng00IlBCtw&list=PL9XzOCngAkqs4m0XdULJu_78nM3Ok3Q65&index=1).